spec/requests/admin/admin_layout_spec.rb

```diff
@@ -0,0 +1,10 @@
+require "rails_helper"
+
+RSpec.describe "Render admin layout", :type => :request do
+
+  it "Must render admin layout, get response" do
+    get "/admin"
+    expect(response)
```

Comment:
this line for what here?

Reply:
maybe you forget to write it to the end? `expect(response).to render_template('index')` or else...